config/v1/0000_10_config-operator_01_dns.crd.yaml

```diff
@@ -14,91 +14,88 @@ spec:
     plural: dnses
     singular: dns
   scope: Cluster
-  preserveUnknownFields: false
   versions:
   - name: v1
     served: true
     storage: true
-  subresources:
```

Comment:
I didn't expect this to pass.  Is your status subresource still present somehow?

Reply:
Hm, looks like I accidentally dropped the `subresources` field (instead of relocating it properly). Strange that this passed.

Reply:
Fixed in `0f55753`. Sorry about that!